src/OpenTK/GameWindow.cs

```diff
@@ -350,9 +382,6 @@ public void Run(double updates_per_second, double frames_per_second)
             }
             finally
             {
-                Move -= DispatchUpdateAndRenderFrame;
```

Comment:
How are these events handled now? Were they even enabled in the first place?

Reply:
It seems like those events used to be registered, but those lines are now commented out: https://github.com/opentk/opentk/blob/19f81efc8976ffe2d5f9a9885cde9f8c021559ec/src/OpenTK/GameWindow.cs#L333 
From git blame, looks like it was commented out 7 years ago, but they forgot to remove the deregistering code, I guess.